README.md

```diff
@@ -2,8 +2,7 @@
 
 # Second project
 ## IMPORTAN! attribution and acknowledgment
-this proyect is the work of (https://github.com/Laylooo) she gave me permition to use it in this time for this practice.
-
+this project is the work of (https://github.com/Laylooo) she gave me permission to use it at this time for this practice.
```

Comment:
Please can you tell me why do you request to use it?

Reply:
Great job using the inline comments. However, providing a 'why' to the request means to write why did you (the reviewer) request this change to the student, not the other way around! 🙏🏻🙏🏻